components/button/README.md

```diff
@@ -60,3 +60,28 @@ The `d2l-button-icon` element can be used just like the native `button`, for ins
 * `icon` (required): Icon for the button (ex. `d2l-tier1:gear`)
 * `h-align` (optional): `text` aligns the leading edge of text
 * `translucent` (optional): Indicates to display translucent (ex. on rich backgrounds)
+
+## d2l-floating-buttons
+
+Floating workflow buttons behavior can be added by using the `<d2l-floating-buttons>` custom element. When the normal position of the workflow buttons is below the bottom edge of the viewport, they will dock at the bottom edge. When the normal position becomes visible, they will undock.
+
+![Floating Buttons](./screenshots/floating-buttons.png?raw=true)
```

Comment:
I think this path needs updating.

Reply:
I think it's correct - it seems to work

Reply:
Oh ok, yeah I guess the path works as relative!